course-v2/layouts/partials/course_detail.html

```diff
@@ -1,15 +1,15 @@
 <div class="card">
   <div class="card-body">
-    <div class="course-detail-section">
+    <section class="course-detail-section">
       {{ partial "course_description.html" (dict "context" .) }}
-    </div>
+    </section>
```

Comment:
⚠️⚠️ Site change! *very minor*

Changing div -> section here made seemed reasonable and made selecting "Course Info" section in Playwright a little easier. See https://github.com/testing-library/dom-testing-library/issues/403#issuecomment-554536248 for a related issue. (I believe that issue is where I picked up the pattern of using `.closest()` for stuff like this. 🤷 )